src/e3/anod/context.py

```diff
@@ -439,7 +439,22 @@ def add_dep(spec_instance: Anod, dep: Dependency, dep_instance: Anod) -> None:
                         spec_instance.name, dep.local_name
                     ),
                 )
-            spec_instance.deps[dep.local_name] = dep_instance
+            if dep.kind != "source":
+                spec_instance.deps[dep.local_name] = dep_instance
+            else:
+                srcbuild_list = dep_instance.source_pkg_build
+                spec_instance.source_list[dep.local_name] = set()
+                if not srcbuild_list:
+                    raise AnodError(
+                        f"{dep.local_name} is marked as source_pkg but not sources is "
```

Comment:
no *source* is defined, ot no sources *are* defined ;)

Reply:
`are` seems better, I will correct this typo :)

Reply:
Done.